### PR TITLE
Setup.sh installs a system service to manage Kuzzle

### PR DESCRIPTION
**Pull request by xbill82**
:warning: Merge after #20

Setup.sh now proposes to the user to install a system-wide service to manage Kuzzle with commands like `service kuzzle start` or `service kuzzle stop`.

### CODE REVIEW

**Comment by xbill82** [comment]
By the way, I have a doubt. In the current implementation, the script first runs Kuzzle and, once it works, it proposes to install the system-wide service.
Regardless of the service, the help provided is always the same and it uses Docker bare-bone. Moreover, since Kuzzle has been started using Docker, I guess the service won't be able to stop it (not sure about that).

I wonder if it wouldn't be better to propose the service _before_ running Kuzzle and if the user accepts, running Kuzzle via the service and provide short help based on this choice.

File: setup.sh [hunk 1]
```diff
@@ -457,11 +488,113 @@ startKuzzle() {
...
 }
 
-isKuzzleRunning() {
-  write "[ℹ] Checking that everything is running"
+installSystemService() {
+  KUZZLE_ETC_PATH="/etc/kuzzle"
+  SYSTEMD_FILE_PATH="/lib/systemd/system/kuzzle.service"
+  if [[ `systemctl` =~ -\.mount ]]; then
+    echo
+    writeBold "[❓] Would you like to install Kuzzle as a system service? (y/N)"
```
Comment (benoitvidis) [inline setup.sh]: s/system/systemd/

File: setup.sh [hunk 2]
```diff
@@ -457,11 +488,113 @@ startKuzzle() {
   done
 }
 
-isKuzzleRunning() {
-  write "[ℹ] Checking that everything is running"
+installSystemService() {
+  KUZZLE_ETC_PATH="/etc/kuzzle"
```
Comment (benoitvidis) [inline setup.sh]: `/etc` is traditionally used to store configuration.

A docker-compose file is more a similar to a launcher to me (i.e. *.desktop files).
Can we use a more generic location? (I suggest `/opt/kuzzle`)

File: setup.sh [hunk 3]
```diff
@@ -457,11 +488,113 @@ startKuzzle() {
...
+WantedBy=multi-user.target
+EndOfMessage
+)
+          $SUDO_PREFIX tee "$SYSTEMD_FILE_PATH" <<< "$SERVICE_FILE_CONTENT" &> /dev/null
+          $SUDO_PREFIX systemctl enable /lib/systemd/system/kuzzle.service
+          if [ $? -eq 0 ]; then
+            echo
+            writeBold "$GREEN" "[✔] Kuzzle service successfully installed."
+            write     "    Use the following commands to manage Kuzzle"
+            write     "    service kuzzle start"
```
Comment (benoitvidis) [inline setup.sh]: `service` is a system V util. It is wrapped to systemctl on debian-like, so it works but is not standard.

`systemctl start|stop|status kuzzle`

**Pull request merged**